src/containers/find-offer/offer-card-square/OfferCardSquare.tsx

```diff
@@ -98,7 +103,12 @@ const OfferCardSquare: FC<OfferCardSquareProps> = ({
       <AppButton sx={styles.sendMessageButton}>
         {t('common.labels.sendMessage')}
       </AppButton>
-      <AppButton fullWidth sx={styles.viewDetailsButton}>
+      <AppButton
```

Comment:
this card should use actions as well as big card

Reply:
fixed